packages/beacon-node/test/spec/presets/index.test.ts

```diff
@@ -22,7 +22,16 @@ import {transition} from "./transition.js";
 const skipOpts: SkipOpts = {
   skippedForks: [],
   skippedRunners: [],
-  skippedHandlers: ["full_withdrawals", "partial_withdrawals", "bls_to_execution_change", "withdrawals"],
+  skippedHandlers: [
+    "full_withdrawals",
+    "partial_withdrawals",
+    "bls_to_execution_change",
+    "withdrawals",
+    "historical_summaries_update",
+  ],
+  // This test right now has been escalted as history summary should have been pushed
+  // in the state, but in the expected state as per test, nothing has been pushed
```

Comment:
escalated the test to the CL group, waiting on revert but seems like lodestar is doing the right thing